File: TestPython/pipelines/DONOT_OPEN_PYTHON/code/pythonorganization/donot/openme12/graph/all_type_main_1/Deduplicate_1_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from pythonorganization.donot.openme12.udfs.UDFs import *

def Deduplicate_1_1_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.dropDuplicates(["c_double"])

File: ScalaProject/sanity-pipeline/pipelines/PERF_SC_PIP/code/src/main/scala/org/main/scla_dep_mgmt_change/graph/PERF_SG/Deduplicate_1_1_1.scala
package org.main.scla_dep_mgmt_change.graph.PERF_SG

import io.prophecy.libs._
import org.main.scla_dep_mgmt_change.udfs.UDFs._
import org.main.scla_dep_mgmt_change.graph.PERF_SG.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Deduplicate_1_1_1 {

  def apply(context: Context, in: DataFrame): DataFrame = {
    import org.apache.spark.sql.expressions.Window
    in.dropDuplicates(
      List("cmls_acct_fundg_srce_cd_drvd",
           "cmls_acct_fundg_srce_cd_ri",
           "cmls_acct_fundg_srce_cd_ri"
      )
    )
  }

}
